trl/experimental/rtpo/rtpo_config.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020-2025 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from ...trainer.grpo_config import GRPOConfig
+
+
+@dataclass
+class RTPOConfig(GRPOConfig):
+    """
+    Configuration class for PAPOTrainer.
+
+    PAPO (Perception-Aware Policy Optimization) extends GRPO/DAPO for multimodal reasoning by adding an implicit
+    perception loss and double entropy regularization.
+
+    Args:
+        schedule_type (`str`,defaults to linear):
+            Choose a schedule type for AnnealingScheduler to control thinking guidance length.
+            Supports: linear, cosine, exponential, piecewise, constant
```

Comment:
```suggestion
        schedule_type (`str`, *optional*, defaults to `"linear"`):
            Choose a schedule type for AnnealingScheduler to control thinking guidance length. Supports: `"linear"`,
            `"cosine, `"exponential"`, `"piecewise"`, `"constant"`.
```

can you try to try to align the docstring with the rest of the codebase? Above is an example